### ISSUE TITLE
Call bumpversion?

### ISSUE THREAD

**Issue by mauritsvanrees** [collective/zestreleaser.towncrier#5]
When there is a `news/X.feature` file, we could automatically call `bumpversion --feature`, and with `news/.Y.breaking` we could call `bumpversion --breaking`.
We would need to read `pyproject.toml` then, to look for `[tool.towncrier] directory = "news"`.
`bumpversion` would *ask* you to confirm, so you can still back out.

**Comment by gforcada** [comment]
`bumpversion` comes from... which package? :thinking: 
**EDIT:** sorry I was so lazy to google: https://pypi.org/project/bumpversion/ then this is yet another dependency right? I see that it does create tags and commits, should that be rather then integrated with zest.releaser directly? As they seem to overlap...

**Comment by mauritsvanrees** [comment]
Nope.  The `bumpversion` command I am talking about is part of `zest.releaser`. :-)
See https://zestreleaser.readthedocs.io/en/latest/overview.html?highlight=bumpversion near the bottom.
And the [bumpversion file in zest.releaser](https://github.com/zestsoftware/zest.releaser/blob/master/zest/releaser/bumpversion.py).

**Comment by gforcada** [comment]
While that is not being integrated, an easy and quick solution would be to print the files on news/ folder before the release number version. That would help a lot.

While I wait for that I use a split terminal to see what's on the news entries and the fullrelease command :smile:
